plugins/UsersManager/lang/en.json

```diff
@@ -8,7 +8,7 @@
         "AnonymousUserHasViewAccess2": "Your analytics reports and your visitors information are publicly viewable.",
         "ApplyToAllWebsites": "Apply to all websites",
         "ChangeAllConfirm": "Are you sure you want to give '%s' access to all the websites?",
-        "ChangePasswordConfirm": "Changing the password will also change the user's token_auth. Do you really want to continue?",
+        "ChangePasswordConfirm": "Changing the password will also change the user's old token_auth. This will only happen once. Do you really want to continue?",
```

Comment:
could we display this modal confirmation only when the old token will be actually changed (ie. the first time a user changes password)? 

(displaying it all the time would be bit confusing + the message mentioned "This will only happen once.")